docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -97,10 +97,10 @@ Time therefore has to be considered slightly fuzzy in DAML, with the fuzziness d
 
 For details, see :ref:`Background concepts - time <time>`.
 
-Time in scenarios
+Time in scripts
 ~~~~~~~~~~~~~~~~~
 
-In scenarios, record and ledger time are always equal. You can set them using the following functions:
+In scripts, record and ledger time are always equal. You can set them using the following functions:
```

Comment:
I guess this is not true anymore for a script running against a real ledger?

Reply:
Oh, good catch, I was just blindly replacing. How about this?
```suggestion
In test scripts, record and ledger time are always equal. You can set them using the following functions:
```